### ISSUE TITLE
gdprCookiesEnabled event not running on page load

### ISSUE THREAD

**Issue by Antimated** [passatgt/gdpr-cookie-notice#31]
Is there a way to trigger the gdprCookiesEnabled event when the user loads the page? As of now the only time this event triggers is when the user accepts the cookies. Is this intentional or am I missing something?

**Comment by passatgt** [comment]
This line should init that event: https://github.com/passatgt/gdpr-cookie-notice/blob/3c48826c0a3badf381d36abb47c77faca1f1a28e/src/js/script.js#L39

**Comment by Antimated** [comment]
Yes that's what I thought, but on demo (https://passatgt.github.io/gdpr-cookie-notice) the event isn't dispatched either. Only when you accept the cookies the event dispatches. Also no cookies are removed if I'm not mistaken. Any ideas? @passatgt

**Comment by RF1705** [comment]
you need to define your Eventistener bevor inluding the Cookie Script
